src/test/java/com/kaltura/client/test/tests/servicesTests/bookmarkTests/AddTests.java

```diff
@@ -32,13 +29,13 @@
 
     @BeforeClass
     private void add_tests_before_class() {
-        // Get asset id from ingest
+        // Get VOD asset id from ingest
         assetId = mediaAsset.getId();
-        type = AssetType.get("MEDIA");
-        AssetReferenceType assetReferenceType = AssetReferenceType.get("MEDIA");
+        type = AssetType.get(AssetType.MEDIA.getValue());
```

Comment:
variable type is useless you can use directly when it needed AssetType.MEDIA etc